src/components/table/column-picker/column-picker.component.ts

```diff
@@ -187,6 +355,50 @@ export class ColumnPickerComponent {
         this.selected = [...selected];
     }
 
+    /** Check if column value or string */
+    isColumnPickerItem(column: ColumnPickerValue): column is ColumnPickerGroupItem {
+        return (column as ColumnPickerGroupItem).name !== undefined;
+    }
+
+    /** Get the column name based on type */
+    getColumnName(item: ColumnPickerValue) {
+        return this.isColumnPickerItem(item) ? item.name : item;
+    }
+
+    /** Check if tree group has visible children */
+    nodeHasChildren(index: number, node: ColumnPickerTreeNode): boolean {
```

Comment:
Any functions that would be private but are being used in the view should be prefixed with an underscore to ensure that no consumer of the component tries to relying on these "private" methods